@aajisaka there is a still a problem with this change - see https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3988/16/artifact/out/patch-mvninstall-root.txt

```
[INFO] Apache Hadoop Client Packaging Invariants for Test . FAILURE [  0.824 s]
```

The issue is this (truncated a bit):
```
[ERROR] Found artifact with unexpected contents: '/home/jenkins/jenkins-agent/workspace/hadoop-multibranch_PR-3988/ubuntu-focal/src/hadoop-client-modules/hadoop-client-minicluster/target/hadoop-client-minicluster-3.4.0-SNAPSHOT.jar'
    Please check the following and either correct the build or update
    the allowed list with reasoning.

    javax/
    javax/xml/
    javax/xml/bind/
    javax/xml/bind/annotation/
    javax/xml/bind/annotation/adapters/
    javax/xml/bind/annotation/adapters/XmlAdapter.class
    javax/xml/bind/annotation/adapters/XmlJavaTypeAdapters.class
    javax/xml/bind/annotation/adapters/XmlJavaTypeAdapter$DEFAULT.class
    javax/xml/bind/annotation/adapters/XmlJavaTypeAdapter.class
    javax/xml/bind/annotation/adapters/CollapsedStringAdapter.class
    javax/xml/bind/annotation/adapters/HexBinaryAdapter.class
    javax/xml/bind/annotation/adapters/NormalizedStringAdapter.class
    javax/xml/bind/annotation/XmlValue.class
    javax/xml/bind/annotation/XmlRegistry.class
    javax/xml/bind/annotation/XmlElements.class
    javax/xml/bind/annotation/XmlElement$DEFAULT.class
    javax/xml/bind/annotation/XmlElement.class
    javax/xml/bind/annotation/XmlSchema.class
    javax/xml/bind/annotation/XmlNs.class
    javax/xml/bind/annotation/XmlNsForm.class
    javax/xml/bind/annotation/XmlType$DEFAULT.class
    javax/xml/bind/annotation/XmlType.class
    javax/xml/bind/annotation/XmlElementRefs.class
    javax/xml/bind/annotation/XmlElementRef$DEFAULT.class
    javax/xml/bind/annotation/XmlElementRef.class
    javax/xml/bind/annotation/XmlElementDecl$GLOBAL.class
    javax/xml/bind/annotation/XmlElementDecl.class
    javax/xml/bind/annotation/XmlElementWrapper.class
    javax/xml/bind/annotation/DomHandler.class
    javax/xml/bind/annotation/XmlMimeType.class
    javax/xml/bind/annotation/XmlSeeAlso.class
    javax/xml/bind/annotation/W3CDomHandler.class
    javax/xml/bind/annotation/XmlIDREF.class
    javax/xml/bind/annotation/XmlAccessType.class
    javax/xml/bind/annotation/XmlAccessorOrder.class
    javax/xml/bind/annotation/XmlAccessOrder.class
    javax/xml/bind/annotation/XmlAttachmentRef.class
    javax/xml/bind/annotation/XmlAnyElement.class
    javax/xml/bind/annotation/XmlSchemaTypes.class
    javax/xml/bind/annotation/XmlSchemaType$DEFAULT.class
    javax/xml/bind/annotation/XmlSchemaType.class
    javax/xml/bind/annotation/XmlRootElement.class
    javax/xml/bind/annotation/XmlAttribute.class
    javax/xml/bind/annotation/XmlMixed.class
    javax/xml/bind/annotation/XmlAccessorType.class
```

[EDIT] I think I fixed this with https://github.com/apache/hadoop/pull/3988/commits/629399a162b2ffb4793131c1300a238224b9e1d6

--- a/hadoop-client-modules/hadoop-client-minicluster/pom.xml
+++ b/hadoop-client-modules/hadoop-client-minicluster/pom.xml
@@ -439,6 +439,10 @@
           <groupId>com.fasterxml.jackson.jaxrs</groupId>
           <artifactId>jackson-jaxrs-json-provider</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>com.sun.xml.bind</groupId>
+          <artifactId>jaxb-impl</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
     <dependency>

